src/test/java/edu/iis/powp/gui/TestPlotSoftPatterns.java

```diff
@@ -42,9 +52,19 @@ private static void setupDrivers(Application application) {
 		application.addDriver("Client Plotter", clientPlotter);
 		application.getDriverManager().setCurrentPlotter(clientPlotter);
 
-		IPlotter plotter = new MyAdapter();
+		IPlotter plotter = new MyAdapter(DrawerFeature.getDrawerController());
+		IPlotter plotter2 = new LinePlotterAdapter(DrawerFeature.getDrawerController());
+		IPlotter plotter3 = (IPlotter) new CommandDrawLineToPosition( DrawerFeature.getDrawerController());
```

Comment:
This is obviously incorrect!!! Commands are not drivers.
PLEASE FIX: The test should be added to TESTS menu (addTest). It should generate and then execute a command.